docs/user_guide.md

```diff
@@ -305,3 +305,61 @@ make deploy
     RESOURCE_PREFIX=<kuberntes-resource-name-prefix> \
     WATCH_NAMESPACE=<namespace-to-watch>
 ```
+
+### Cancel running Flink job
+
+If you want to cancel a running Flink job, attach control annotation to your FlinkCluster's metadata:
+```
+metadata:
+  annotations:
+    flinkclusters.flinkoperator.k8s.io/desired-control: job-cancel
+```
+
+When canceling, all pods that make up the Flink cluster are basically terminated.
+If you want to leave the cluster, configure spec.job.cleanupPolicy.afterJobCancelled according to [FlinkCluster CRD doc](../docs/crd.md)
+
+When job cancellation is finished, the control annotation disappears and the progress can be checked in FlinkCluster status:
+```bash
+kubectl describe flinkcluster <CLUSTER-NAME>
+
+...
+
+Status:
+  Control:
+    Data:            <nil>
+    Name:            job-cancel
+    State:           Succeeded
+    Update Time:     2020-04-03T10:04:50+09:00
+```
+
+### Create savepoint
```

Comment:
We have a separate doc for managing savepoints, could you add this section there? I might merge the doc to the user guide in the future.

https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/blob/master/docs/savepoints_guide.md

Reply:
I have added to the guide.